CRITICAL - 2021-01-02 13:54:33 --> Trying to get property 'rfc_responsable' of non-object
#0 C:\xampp\htdocs\comple\app\Controllers\Responsable\AsistenciaController.php(21): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get p...', 'C:\\xampp\\htdocs...', 21, Array)
#1 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\AsistenciaController->listaAsistencia()
#2 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\AsistenciaController))
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2021-01-02 13:54:40 --> Trying to get property 'rfc_responsable' of non-object
#0 C:\xampp\htdocs\comple\app\Controllers\Responsable\InicioController.php(17): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get p...', 'C:\\xampp\\htdocs...', 17, Array)
#1 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\InicioController->index()
#2 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\InicioController))
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2021-01-02 13:59:14 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `insc`.`...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Responsable\AsistenciaModel.php(21): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Models\Responsable\AsistenciaModel.php(55): App\Models\Responsable\AsistenciaModel->get_inscripciones('12', 1)
#4 C:\xampp\htdocs\comple\app\Services\Responsable\AsistenciaService.php(20): App\Models\Responsable\AsistenciaModel->get_actividad_alumno('12', 1)
#5 C:\xampp\htdocs\comple\app\Controllers\Responsable\AsistenciaController.php(77): App\Services\Responsable\AsistenciaService->get_actividad_alumno('12', 1)
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\AsistenciaController->listaCalificacion()
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\AsistenciaController))
#8 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2021-01-02 14:08:46 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `insc`.`...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Responsable\AsistenciaModel.php(21): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Models\Responsable\AsistenciaModel.php(55): App\Models\Responsable\AsistenciaModel->get_inscripciones('12', 1)
#4 C:\xampp\htdocs\comple\app\Services\Responsable\AsistenciaService.php(20): App\Models\Responsable\AsistenciaModel->get_actividad_alumno('12', 1)
#5 C:\xampp\htdocs\comple\app\Controllers\Responsable\AsistenciaController.php(77): App\Services\Responsable\AsistenciaService->get_actividad_alumno('12', 1)
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\AsistenciaController->listaCalificacion()
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\AsistenciaController))
#8 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2021-01-02 14:10:27 --> Undefined variable: fecha
#0 C:\xampp\htdocs\comple\app\Views\Responsable\Lista-Calificacion\listar.php(110): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 110, Array)
#1 C:\xampp\htdocs\comple\system\View\View.php(246): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\comple\system\Common.php(1119): CodeIgniter\View\View->render('Responsable/Lis...', Array, true)
#3 C:\xampp\htdocs\comple\app\Controllers\Responsable\AsistenciaController.php(88): view('Responsable/Lis...', Array)
#4 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\AsistenciaController->listaCalificacion()
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\AsistenciaController))
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2021-01-02 14:13:45 --> Undefined property: stdClass::$nombre_responsable
#0 C:\xampp\htdocs\comple\app\Views\Responsable\Lista-Calificacion\listar.php(121): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 121, Array)
#1 C:\xampp\htdocs\comple\system\View\View.php(246): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\comple\system\Common.php(1119): CodeIgniter\View\View->render('Responsable/Lis...', Array, true)
#3 C:\xampp\htdocs\comple\app\Controllers\Responsable\AsistenciaController.php(88): view('Responsable/Lis...', Array)
#4 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\AsistenciaController->listaCalificacion()
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\AsistenciaController))
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2021-01-02 14:26:59 --> Undefined property: stdClass::$nombre
#0 C:\xampp\htdocs\comple\app\Models\Responsable\AsistenciaModel.php(74): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 74, Array)
#1 C:\xampp\htdocs\comple\app\Services\Responsable\AsistenciaService.php(20): App\Models\Responsable\AsistenciaModel->get_actividad_alumno('12', '1')
#2 C:\xampp\htdocs\comple\app\Controllers\Responsable\AsistenciaController.php(77): App\Services\Responsable\AsistenciaService->get_actividad_alumno('12', 1)
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\AsistenciaController->listaCalificacion()
#4 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\AsistenciaController))
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2021-01-02 14:27:59 --> Undefined property: stdClass::$nombre_responsable
#0 C:\xampp\htdocs\comple\app\Models\Responsable\AsistenciaModel.php(74): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 74, Array)
#1 C:\xampp\htdocs\comple\app\Services\Responsable\AsistenciaService.php(20): App\Models\Responsable\AsistenciaModel->get_actividad_alumno('12', '1')
#2 C:\xampp\htdocs\comple\app\Controllers\Responsable\AsistenciaController.php(77): App\Services\Responsable\AsistenciaService->get_actividad_alumno('12', 1)
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\AsistenciaController->listaCalificacion()
#4 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\AsistenciaController))
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2021-01-02 14:28:27 --> Undefined property: stdClass::$nombre
#0 C:\xampp\htdocs\comple\app\Models\Responsable\AsistenciaModel.php(74): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 74, Array)
#1 C:\xampp\htdocs\comple\app\Services\Responsable\AsistenciaService.php(20): App\Models\Responsable\AsistenciaModel->get_actividad_alumno('12', '1')
#2 C:\xampp\htdocs\comple\app\Controllers\Responsable\AsistenciaController.php(77): App\Services\Responsable\AsistenciaService->get_actividad_alumno('12', 1)
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\AsistenciaController->listaCalificacion()
#4 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\AsistenciaController))
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2021-01-02 14:28:58 --> Undefined property: stdClass::$nombre
#0 C:\xampp\htdocs\comple\app\Models\Responsable\AsistenciaModel.php(74): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 74, Array)
#1 C:\xampp\htdocs\comple\app\Services\Responsable\AsistenciaService.php(20): App\Models\Responsable\AsistenciaModel->get_actividad_alumno('12', '1')
#2 C:\xampp\htdocs\comple\app\Controllers\Responsable\AsistenciaController.php(77): App\Services\Responsable\AsistenciaService->get_actividad_alumno('12', 1)
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\AsistenciaController->listaCalificacion()
#4 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\AsistenciaController))
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2021-01-02 14:29:28 --> Undefined property: stdClass::$nombre
#0 C:\xampp\htdocs\comple\app\Models\Responsable\AsistenciaModel.php(74): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 74, Array)
#1 C:\xampp\htdocs\comple\app\Services\Responsable\AsistenciaService.php(20): App\Models\Responsable\AsistenciaModel->get_actividad_alumno('12', '1')
#2 C:\xampp\htdocs\comple\app\Controllers\Responsable\AsistenciaController.php(77): App\Services\Responsable\AsistenciaService->get_actividad_alumno('12', 1)
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\AsistenciaController->listaCalificacion()
#4 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\AsistenciaController))
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2021-01-02 14:30:05 --> Undefined property: stdClass::$nombre
#0 C:\xampp\htdocs\comple\app\Models\Responsable\AsistenciaModel.php(74): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 74, Array)
#1 C:\xampp\htdocs\comple\app\Services\Responsable\AsistenciaService.php(20): App\Models\Responsable\AsistenciaModel->get_actividad_alumno('12', '1')
#2 C:\xampp\htdocs\comple\app\Controllers\Responsable\AsistenciaController.php(77): App\Services\Responsable\AsistenciaService->get_actividad_alumno('12', 1)
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\AsistenciaController->listaCalificacion()
#4 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\AsistenciaController))
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2021-01-02 14:30:44 --> Undefined property: stdClass::$nombre
#0 C:\xampp\htdocs\comple\app\Models\Responsable\AsistenciaModel.php(74): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 74, Array)
#1 C:\xampp\htdocs\comple\app\Services\Responsable\AsistenciaService.php(20): App\Models\Responsable\AsistenciaModel->get_actividad_alumno('12', '1')
#2 C:\xampp\htdocs\comple\app\Controllers\Responsable\AsistenciaController.php(77): App\Services\Responsable\AsistenciaService->get_actividad_alumno('12', 1)
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\AsistenciaController->listaCalificacion()
#4 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\AsistenciaController))
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2021-01-02 14:31:45 --> Undefined property: stdClass::$nombre
#0 C:\xampp\htdocs\comple\app\Models\Responsable\AsistenciaModel.php(74): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 74, Array)
#1 C:\xampp\htdocs\comple\app\Services\Responsable\AsistenciaService.php(20): App\Models\Responsable\AsistenciaModel->get_actividad_alumno('12', '1')
#2 C:\xampp\htdocs\comple\app\Controllers\Responsable\AsistenciaController.php(77): App\Services\Responsable\AsistenciaService->get_actividad_alumno('12', 1)
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\AsistenciaController->listaCalificacion()
#4 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\AsistenciaController))
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2021-01-02 14:35:05 --> Undefined property: stdClass::$nombre
#0 C:\xampp\htdocs\comple\app\Models\Responsable\AsistenciaModel.php(74): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 74, Array)
#1 C:\xampp\htdocs\comple\app\Services\Responsable\AsistenciaService.php(20): App\Models\Responsable\AsistenciaModel->get_actividad_alumno('12', '1')
#2 C:\xampp\htdocs\comple\app\Controllers\Responsable\AsistenciaController.php(77): App\Services\Responsable\AsistenciaService->get_actividad_alumno('12', 1)
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\AsistenciaController->listaCalificacion()
#4 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\AsistenciaController))
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2021-01-02 14:36:39 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Responsable\AsistenciaModel.php(31): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Models\Responsable\AsistenciaModel.php(62): App\Models\Responsable\AsistenciaModel->get_alumno('201160074')
#4 C:\xampp\htdocs\comple\app\Services\Responsable\AsistenciaService.php(20): App\Models\Responsable\AsistenciaModel->get_actividad_alumno('12', '1')
#5 C:\xampp\htdocs\comple\app\Controllers\Responsable\AsistenciaController.php(77): App\Services\Responsable\AsistenciaService->get_actividad_alumno('12', 1)
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\AsistenciaController->listaCalificacion()
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\AsistenciaController))
#8 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2021-01-02 14:36:55 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `insc`.`...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Responsable\AsistenciaModel.php(22): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Models\Responsable\AsistenciaModel.php(56): App\Models\Responsable\AsistenciaModel->get_inscripciones('12', 1)
#4 C:\xampp\htdocs\comple\app\Services\Responsable\AsistenciaService.php(20): App\Models\Responsable\AsistenciaModel->get_actividad_alumno('12', 1)
#5 C:\xampp\htdocs\comple\app\Controllers\Responsable\AsistenciaController.php(77): App\Services\Responsable\AsistenciaService->get_actividad_alumno('12', 1)
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\AsistenciaController->listaCalificacion()
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\AsistenciaController))
#8 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2021-01-02 14:37:06 --> Undefined property: stdClass::$nombre
#0 C:\xampp\htdocs\comple\app\Models\Responsable\AsistenciaModel.php(74): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 74, Array)
#1 C:\xampp\htdocs\comple\app\Services\Responsable\AsistenciaService.php(20): App\Models\Responsable\AsistenciaModel->get_actividad_alumno('12', '1')
#2 C:\xampp\htdocs\comple\app\Controllers\Responsable\AsistenciaController.php(77): App\Services\Responsable\AsistenciaService->get_actividad_alumno('12', 1)
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\AsistenciaController->listaCalificacion()
#4 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\AsistenciaController))
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2021-01-02 14:37:22 --> Undefined property: stdClass::$nombre
#0 C:\xampp\htdocs\comple\app\Models\Responsable\AsistenciaModel.php(74): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 74, Array)
#1 C:\xampp\htdocs\comple\app\Services\Responsable\AsistenciaService.php(20): App\Models\Responsable\AsistenciaModel->get_actividad_alumno('12', '1')
#2 C:\xampp\htdocs\comple\app\Controllers\Responsable\AsistenciaController.php(77): App\Services\Responsable\AsistenciaService->get_actividad_alumno('12', 1)
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\AsistenciaController->listaCalificacion()
#4 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\AsistenciaController))
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2021-01-02 14:38:33 --> Undefined property: stdClass::$nombre
#0 C:\xampp\htdocs\comple\app\Models\Responsable\AsistenciaModel.php(74): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 74, Array)
#1 C:\xampp\htdocs\comple\app\Services\Responsable\AsistenciaService.php(20): App\Models\Responsable\AsistenciaModel->get_actividad_alumno('12', '1')
#2 C:\xampp\htdocs\comple\app\Controllers\Responsable\AsistenciaController.php(77): App\Services\Responsable\AsistenciaService->get_actividad_alumno('12', 1)
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\AsistenciaController->listaCalificacion()
#4 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\AsistenciaController))
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2021-01-02 14:40:13 --> Undefined property: stdClass::$apaterno
#0 C:\xampp\htdocs\comple\app\Models\Responsable\AsistenciaModel.php(74): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 74, Array)
#1 C:\xampp\htdocs\comple\app\Services\Responsable\AsistenciaService.php(20): App\Models\Responsable\AsistenciaModel->get_actividad_alumno('12', '1')
#2 C:\xampp\htdocs\comple\app\Controllers\Responsable\AsistenciaController.php(77): App\Services\Responsable\AsistenciaService->get_actividad_alumno('12', 1)
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\AsistenciaController->listaCalificacion()
#4 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\AsistenciaController))
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2021-01-02 14:41:55 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `insc`.`...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Responsable\AsistenciaModel.php(21): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Models\Responsable\AsistenciaModel.php(56): App\Models\Responsable\AsistenciaModel->get_inscripciones('12', 1)
#4 C:\xampp\htdocs\comple\app\Services\Responsable\AsistenciaService.php(20): App\Models\Responsable\AsistenciaModel->get_actividad_alumno('12', 1)
#5 C:\xampp\htdocs\comple\app\Controllers\Responsable\AsistenciaController.php(77): App\Services\Responsable\AsistenciaService->get_actividad_alumno('12', 1)
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\AsistenciaController->listaCalificacion()
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\AsistenciaController))
#8 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2021-01-02 14:42:08 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `insc`.`...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Responsable\AsistenciaModel.php(21): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Models\Responsable\AsistenciaModel.php(56): App\Models\Responsable\AsistenciaModel->get_inscripciones('12', 1)
#4 C:\xampp\htdocs\comple\app\Services\Responsable\AsistenciaService.php(20): App\Models\Responsable\AsistenciaModel->get_actividad_alumno('12', 1)
#5 C:\xampp\htdocs\comple\app\Controllers\Responsable\AsistenciaController.php(77): App\Services\Responsable\AsistenciaService->get_actividad_alumno('12', 1)
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\AsistenciaController->listaCalificacion()
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\AsistenciaController))
#8 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2021-01-02 14:42:22 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `insc`.`...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Responsable\AsistenciaModel.php(21): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Models\Responsable\AsistenciaModel.php(56): App\Models\Responsable\AsistenciaModel->get_inscripciones('12', 1)
#4 C:\xampp\htdocs\comple\app\Services\Responsable\AsistenciaService.php(20): App\Models\Responsable\AsistenciaModel->get_actividad_alumno('12', 1)
#5 C:\xampp\htdocs\comple\app\Controllers\Responsable\AsistenciaController.php(77): App\Services\Responsable\AsistenciaService->get_actividad_alumno('12', 1)
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\AsistenciaController->listaCalificacion()
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\AsistenciaController))
#8 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2021-01-02 14:42:35 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `insc`.`...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Responsable\AsistenciaModel.php(21): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Models\Responsable\AsistenciaModel.php(56): App\Models\Responsable\AsistenciaModel->get_inscripciones('12', 1)
#4 C:\xampp\htdocs\comple\app\Services\Responsable\AsistenciaService.php(20): App\Models\Responsable\AsistenciaModel->get_actividad_alumno('12', 1)
#5 C:\xampp\htdocs\comple\app\Controllers\Responsable\AsistenciaController.php(77): App\Services\Responsable\AsistenciaService->get_actividad_alumno('12', 1)
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\AsistenciaController->listaCalificacion()
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\AsistenciaController))
#8 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2021-01-02 14:43:14 --> Undefined property: stdClass::$nombre
#0 C:\xampp\htdocs\comple\app\Views\Responsable\Lista-Calificacion\listar.php(121): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 121, Array)
#1 C:\xampp\htdocs\comple\system\View\View.php(246): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\comple\system\Common.php(1119): CodeIgniter\View\View->render('Responsable/Lis...', Array, true)
#3 C:\xampp\htdocs\comple\app\Controllers\Responsable\AsistenciaController.php(88): view('Responsable/Lis...', Array)
#4 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\AsistenciaController->listaCalificacion()
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\AsistenciaController))
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2021-01-02 14:44:09 --> Undefined variable: ar
#0 C:\xampp\htdocs\comple\app\Views\Responsable\Lista-Calificacion\listar.php(122): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 122, Array)
#1 C:\xampp\htdocs\comple\system\View\View.php(246): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\comple\system\Common.php(1119): CodeIgniter\View\View->render('Responsable/Lis...', Array, true)
#3 C:\xampp\htdocs\comple\app\Controllers\Responsable\AsistenciaController.php(88): view('Responsable/Lis...', Array)
#4 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\AsistenciaController->listaCalificacion()
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\AsistenciaController))
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2021-01-02 14:48:04 --> Undefined property: stdClass::$nombre_jefe
#0 C:\xampp\htdocs\comple\app\Views\Responsable\Lista-Calificacion\listar.php(122): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 122, Array)
#1 C:\xampp\htdocs\comple\system\View\View.php(246): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\comple\system\Common.php(1119): CodeIgniter\View\View->render('Responsable/Lis...', Array, true)
#3 C:\xampp\htdocs\comple\app\Controllers\Responsable\AsistenciaController.php(88): view('Responsable/Lis...', Array)
#4 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\AsistenciaController->listaCalificacion()
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\AsistenciaController))
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2021-01-02 14:49:40 --> Undefined property: stdClass::$nombre_jefe
#0 C:\xampp\htdocs\comple\app\Views\Responsable\Lista-Calificacion\listar.php(122): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 122, Array)
#1 C:\xampp\htdocs\comple\system\View\View.php(246): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\comple\system\Common.php(1119): CodeIgniter\View\View->render('Responsable/Lis...', Array, true)
#3 C:\xampp\htdocs\comple\app\Controllers\Responsable\AsistenciaController.php(88): view('Responsable/Lis...', Array)
#4 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\AsistenciaController->listaCalificacion()
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\AsistenciaController))
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2021-01-02 14:52:41 --> Maximum execution time of 120 seconds exceeded
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2021-01-02 14:54:26 --> Undefined property: stdClass::$apaterno_jefe
#0 C:\xampp\htdocs\comple\app\Views\Responsable\Lista-Calificacion\listar.php(122): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 122, Array)
#1 C:\xampp\htdocs\comple\system\View\View.php(246): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\comple\system\Common.php(1119): CodeIgniter\View\View->render('Responsable/Lis...', Array, true)
#3 C:\xampp\htdocs\comple\app\Controllers\Responsable\AsistenciaController.php(88): view('Responsable/Lis...', Array)
#4 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\AsistenciaController->listaCalificacion()
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\AsistenciaController))
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2021-01-02 14:57:38 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Responsable\AsistenciaModel.php(30): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Models\Responsable\AsistenciaModel.php(62): App\Models\Responsable\AsistenciaModel->get_alumno('201160126')
#4 C:\xampp\htdocs\comple\app\Services\Responsable\AsistenciaService.php(20): App\Models\Responsable\AsistenciaModel->get_actividad_alumno('2', '1')
#5 C:\xampp\htdocs\comple\app\Controllers\Responsable\AsistenciaController.php(24): App\Services\Responsable\AsistenciaService->get_actividad_alumno('2', true)
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\AsistenciaController->listaAsistencia()
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\AsistenciaController))
#8 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2021-01-02 15:57:23 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `a`.`id_...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Responsable\InicioModel.php(22): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Services\Responsable\InicioService.php(20): App\Models\Responsable\InicioModel->getActividadesPorResponsable('RSMLUISAR4132', true)
#4 C:\xampp\htdocs\comple\app\Controllers\Responsable\InicioController.php(19): App\Services\Responsable\InicioService->getActividadesPorResponsable('RSMLUISAR4132', true)
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\InicioController->index()
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\InicioController))
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2021-01-02 16:06:35 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `insc`.`...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Responsable\AsistenciaModel.php(21): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Models\Responsable\AsistenciaModel.php(56): App\Models\Responsable\AsistenciaModel->get_inscripciones('11', true)
#4 C:\xampp\htdocs\comple\app\Services\Responsable\AsistenciaService.php(20): App\Models\Responsable\AsistenciaModel->get_actividad_alumno('11', true)
#5 C:\xampp\htdocs\comple\app\Controllers\Responsable\AsistenciaController.php(24): App\Services\Responsable\AsistenciaService->get_actividad_alumno('11', true)
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\AsistenciaController->listaAsistencia()
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\AsistenciaController))
#8 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2021-01-02 16:06:49 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `insc`.`...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Responsable\AsistenciaModel.php(21): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Models\Responsable\AsistenciaModel.php(56): App\Models\Responsable\AsistenciaModel->get_inscripciones('11', true)
#4 C:\xampp\htdocs\comple\app\Services\Responsable\AsistenciaService.php(20): App\Models\Responsable\AsistenciaModel->get_actividad_alumno('11', true)
#5 C:\xampp\htdocs\comple\app\Controllers\Responsable\AsistenciaController.php(24): App\Services\Responsable\AsistenciaService->get_actividad_alumno('11', true)
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\AsistenciaController->listaAsistencia()
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\AsistenciaController))
#8 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2021-01-02 22:44:15 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `r`.`rfc...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\ResponsableLoginModel.php(21): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Services\ResponsableLoginService.php(22): App\Models\ResponsableLoginModel->getResponsablePorResponsable('SSADG123GH12E')
#4 C:\xampp\htdocs\comple\app\Controllers\ResponsableLoginController.php(33): App\Services\ResponsableLoginService->login('SSADG123GH12E', 'sanchezasr2197y...')
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\ResponsableLoginController->login()
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ResponsableLoginController))
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2021-01-02 22:45:02 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `insc`.`...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Responsable\CalificacionModel.php(20): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Models\Responsable\CalificacionModel.php(51): App\Models\Responsable\CalificacionModel->get_inscripciones('11')
#4 C:\xampp\htdocs\comple\app\Services\Responsable\CalificacionService.php(16): App\Models\Responsable\CalificacionModel->get_actividad_alumno('11')
#5 C:\xampp\htdocs\comple\app\Controllers\Responsable\CalificacionController.php(16): App\Services\Responsable\CalificacionService->get_actividad_alumno('11')
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\CalificacionController->index()
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\CalificacionController))
#8 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2021-01-02 23:03:22 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `insc`.`...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Responsable\CalificacionModel.php(20): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Models\Responsable\CalificacionModel.php(51): App\Models\Responsable\CalificacionModel->get_inscripciones('11')
#4 C:\xampp\htdocs\comple\app\Services\Responsable\CalificacionService.php(16): App\Models\Responsable\CalificacionModel->get_actividad_alumno('11')
#5 C:\xampp\htdocs\comple\app\Controllers\Responsable\CalificacionController.php(16): App\Services\Responsable\CalificacionService->get_actividad_alumno('11')
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\CalificacionController->index()
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\CalificacionController))
#8 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2021-01-02 23:03:41 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `insc`.`...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Responsable\CalificacionModel.php(20): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Models\Responsable\CalificacionModel.php(51): App\Models\Responsable\CalificacionModel->get_inscripciones('11')
#4 C:\xampp\htdocs\comple\app\Services\Responsable\CalificacionService.php(16): App\Models\Responsable\CalificacionModel->get_actividad_alumno('11')
#5 C:\xampp\htdocs\comple\app\Controllers\Responsable\CalificacionController.php(16): App\Services\Responsable\CalificacionService->get_actividad_alumno('11')
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\CalificacionController->index()
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\CalificacionController))
#8 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2021-01-02 23:03:47 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `insc`.`...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Responsable\CalificacionModel.php(20): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Models\Responsable\CalificacionModel.php(51): App\Models\Responsable\CalificacionModel->get_inscripciones('11')
#4 C:\xampp\htdocs\comple\app\Services\Responsable\CalificacionService.php(16): App\Models\Responsable\CalificacionModel->get_actividad_alumno('11')
#5 C:\xampp\htdocs\comple\app\Controllers\Responsable\CalificacionController.php(16): App\Services\Responsable\CalificacionService->get_actividad_alumno('11')
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\CalificacionController->index()
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\CalificacionController))
#8 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2021-01-02 23:03:50 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `insc`.`...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Responsable\CalificacionModel.php(20): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Models\Responsable\CalificacionModel.php(51): App\Models\Responsable\CalificacionModel->get_inscripciones('11')
#4 C:\xampp\htdocs\comple\app\Services\Responsable\CalificacionService.php(16): App\Models\Responsable\CalificacionModel->get_actividad_alumno('11')
#5 C:\xampp\htdocs\comple\app\Controllers\Responsable\CalificacionController.php(16): App\Services\Responsable\CalificacionService->get_actividad_alumno('11')
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\CalificacionController->index()
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\CalificacionController))
#8 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2021-01-02 23:03:53 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `insc`.`...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Responsable\CalificacionModel.php(20): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Models\Responsable\CalificacionModel.php(51): App\Models\Responsable\CalificacionModel->get_inscripciones('11')
#4 C:\xampp\htdocs\comple\app\Services\Responsable\CalificacionService.php(16): App\Models\Responsable\CalificacionModel->get_actividad_alumno('11')
#5 C:\xampp\htdocs\comple\app\Controllers\Responsable\CalificacionController.php(16): App\Services\Responsable\CalificacionService->get_actividad_alumno('11')
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\CalificacionController->index()
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\CalificacionController))
#8 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2021-01-02 23:04:02 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `insc`.`...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Responsable\CalificacionModel.php(20): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Models\Responsable\CalificacionModel.php(51): App\Models\Responsable\CalificacionModel->get_inscripciones('11')
#4 C:\xampp\htdocs\comple\app\Services\Responsable\CalificacionService.php(16): App\Models\Responsable\CalificacionModel->get_actividad_alumno('11')
#5 C:\xampp\htdocs\comple\app\Controllers\Responsable\CalificacionController.php(16): App\Services\Responsable\CalificacionService->get_actividad_alumno('11')
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\CalificacionController->index()
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\CalificacionController))
#8 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2021-01-02 23:04:05 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `insc`.`...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Responsable\CalificacionModel.php(20): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Models\Responsable\CalificacionModel.php(51): App\Models\Responsable\CalificacionModel->get_inscripciones('11')
#4 C:\xampp\htdocs\comple\app\Services\Responsable\CalificacionService.php(16): App\Models\Responsable\CalificacionModel->get_actividad_alumno('11')
#5 C:\xampp\htdocs\comple\app\Controllers\Responsable\CalificacionController.php(16): App\Services\Responsable\CalificacionService->get_actividad_alumno('11')
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\CalificacionController->index()
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\CalificacionController))
#8 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2021-01-02 23:04:21 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `insc`.`...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Responsable\CalificacionModel.php(20): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Models\Responsable\CalificacionModel.php(51): App\Models\Responsable\CalificacionModel->get_inscripciones('11')
#4 C:\xampp\htdocs\comple\app\Services\Responsable\CalificacionService.php(16): App\Models\Responsable\CalificacionModel->get_actividad_alumno('11')
#5 C:\xampp\htdocs\comple\app\Controllers\Responsable\CalificacionController.php(16): App\Services\Responsable\CalificacionService->get_actividad_alumno('11')
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\CalificacionController->index()
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\CalificacionController))
#8 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2021-01-02 23:04:24 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `insc`.`...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Responsable\CalificacionModel.php(20): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Models\Responsable\CalificacionModel.php(51): App\Models\Responsable\CalificacionModel->get_inscripciones('11')
#4 C:\xampp\htdocs\comple\app\Services\Responsable\CalificacionService.php(16): App\Models\Responsable\CalificacionModel->get_actividad_alumno('11')
#5 C:\xampp\htdocs\comple\app\Controllers\Responsable\CalificacionController.php(16): App\Services\Responsable\CalificacionService->get_actividad_alumno('11')
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\CalificacionController->index()
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\CalificacionController))
#8 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2021-01-02 23:04:39 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `insc`.`...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Responsable\CalificacionModel.php(20): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Models\Responsable\CalificacionModel.php(51): App\Models\Responsable\CalificacionModel->get_inscripciones('11')
#4 C:\xampp\htdocs\comple\app\Services\Responsable\CalificacionService.php(16): App\Models\Responsable\CalificacionModel->get_actividad_alumno('11')
#5 C:\xampp\htdocs\comple\app\Controllers\Responsable\CalificacionController.php(16): App\Services\Responsable\CalificacionService->get_actividad_alumno('11')
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\CalificacionController->index()
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\CalificacionController))
#8 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2021-01-02 23:04:43 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `insc`.`...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Responsable\CalificacionModel.php(20): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Models\Responsable\CalificacionModel.php(51): App\Models\Responsable\CalificacionModel->get_inscripciones('11')
#4 C:\xampp\htdocs\comple\app\Services\Responsable\CalificacionService.php(16): App\Models\Responsable\CalificacionModel->get_actividad_alumno('11')
#5 C:\xampp\htdocs\comple\app\Controllers\Responsable\CalificacionController.php(16): App\Services\Responsable\CalificacionService->get_actividad_alumno('11')
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\CalificacionController->index()
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\CalificacionController))
#8 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2021-01-02 23:04:45 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `insc`.`...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Responsable\CalificacionModel.php(20): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Models\Responsable\CalificacionModel.php(51): App\Models\Responsable\CalificacionModel->get_inscripciones('11')
#4 C:\xampp\htdocs\comple\app\Services\Responsable\CalificacionService.php(16): App\Models\Responsable\CalificacionModel->get_actividad_alumno('11')
#5 C:\xampp\htdocs\comple\app\Controllers\Responsable\CalificacionController.php(16): App\Services\Responsable\CalificacionService->get_actividad_alumno('11')
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\CalificacionController->index()
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\CalificacionController))
#8 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2021-01-02 23:04:49 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `insc`.`...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Responsable\CalificacionModel.php(20): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Models\Responsable\CalificacionModel.php(51): App\Models\Responsable\CalificacionModel->get_inscripciones('11')
#4 C:\xampp\htdocs\comple\app\Services\Responsable\CalificacionService.php(16): App\Models\Responsable\CalificacionModel->get_actividad_alumno('11')
#5 C:\xampp\htdocs\comple\app\Controllers\Responsable\CalificacionController.php(16): App\Services\Responsable\CalificacionService->get_actividad_alumno('11')
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\CalificacionController->index()
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\CalificacionController))
#8 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2021-01-02 23:04:52 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `insc`.`...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Responsable\CalificacionModel.php(20): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Models\Responsable\CalificacionModel.php(51): App\Models\Responsable\CalificacionModel->get_inscripciones('11')
#4 C:\xampp\htdocs\comple\app\Services\Responsable\CalificacionService.php(16): App\Models\Responsable\CalificacionModel->get_actividad_alumno('11')
#5 C:\xampp\htdocs\comple\app\Controllers\Responsable\CalificacionController.php(16): App\Services\Responsable\CalificacionService->get_actividad_alumno('11')
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\CalificacionController->index()
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\CalificacionController))
#8 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2021-01-02 23:04:58 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `insc`.`...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Responsable\CalificacionModel.php(20): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Models\Responsable\CalificacionModel.php(51): App\Models\Responsable\CalificacionModel->get_inscripciones('11')
#4 C:\xampp\htdocs\comple\app\Services\Responsable\CalificacionService.php(16): App\Models\Responsable\CalificacionModel->get_actividad_alumno('11')
#5 C:\xampp\htdocs\comple\app\Controllers\Responsable\CalificacionController.php(16): App\Services\Responsable\CalificacionService->get_actividad_alumno('11')
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\CalificacionController->index()
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\CalificacionController))
#8 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2021-01-02 23:55:58 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `insc`.`...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Responsable\CalificacionModel.php(20): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Models\Responsable\CalificacionModel.php(51): App\Models\Responsable\CalificacionModel->get_inscripciones('11')
#4 C:\xampp\htdocs\comple\app\Services\Responsable\CalificacionService.php(16): App\Models\Responsable\CalificacionModel->get_actividad_alumno('11')
#5 C:\xampp\htdocs\comple\app\Controllers\Responsable\CalificacionController.php(16): App\Services\Responsable\CalificacionService->get_actividad_alumno('11')
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\CalificacionController->index()
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\CalificacionController))
#8 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2021-01-02 23:56:17 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `insc`.`...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Responsable\CalificacionModel.php(20): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Models\Responsable\CalificacionModel.php(51): App\Models\Responsable\CalificacionModel->get_inscripciones('11')
#4 C:\xampp\htdocs\comple\app\Services\Responsable\CalificacionService.php(16): App\Models\Responsable\CalificacionModel->get_actividad_alumno('11')
#5 C:\xampp\htdocs\comple\app\Controllers\Responsable\CalificacionController.php(16): App\Services\Responsable\CalificacionService->get_actividad_alumno('11')
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\CalificacionController->index()
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\CalificacionController))
#8 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2021-01-02 23:56:52 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `insc`.`...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Responsable\CalificacionModel.php(20): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Models\Responsable\CalificacionModel.php(51): App\Models\Responsable\CalificacionModel->get_inscripciones('11')
#4 C:\xampp\htdocs\comple\app\Services\Responsable\CalificacionService.php(16): App\Models\Responsable\CalificacionModel->get_actividad_alumno('11')
#5 C:\xampp\htdocs\comple\app\Controllers\Responsable\CalificacionController.php(16): App\Services\Responsable\CalificacionService->get_actividad_alumno('11')
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\CalificacionController->index()
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\CalificacionController))
#8 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2021-01-02 23:57:23 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `insc`.`...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Responsable\CalificacionModel.php(20): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Models\Responsable\CalificacionModel.php(51): App\Models\Responsable\CalificacionModel->get_inscripciones('11')
#4 C:\xampp\htdocs\comple\app\Services\Responsable\CalificacionService.php(16): App\Models\Responsable\CalificacionModel->get_actividad_alumno('11')
#5 C:\xampp\htdocs\comple\app\Controllers\Responsable\CalificacionController.php(16): App\Services\Responsable\CalificacionService->get_actividad_alumno('11')
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\CalificacionController->index()
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\CalificacionController))
#8 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2021-01-02 23:57:30 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `insc`.`...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Responsable\CalificacionModel.php(20): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Models\Responsable\CalificacionModel.php(51): App\Models\Responsable\CalificacionModel->get_inscripciones('11')
#4 C:\xampp\htdocs\comple\app\Services\Responsable\CalificacionService.php(16): App\Models\Responsable\CalificacionModel->get_actividad_alumno('11')
#5 C:\xampp\htdocs\comple\app\Controllers\Responsable\CalificacionController.php(16): App\Services\Responsable\CalificacionService->get_actividad_alumno('11')
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\CalificacionController->index()
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\CalificacionController))
#8 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2021-01-02 23:57:45 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `insc`.`...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Responsable\CalificacionModel.php(20): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Models\Responsable\CalificacionModel.php(51): App\Models\Responsable\CalificacionModel->get_inscripciones('11')
#4 C:\xampp\htdocs\comple\app\Services\Responsable\CalificacionService.php(16): App\Models\Responsable\CalificacionModel->get_actividad_alumno('11')
#5 C:\xampp\htdocs\comple\app\Controllers\Responsable\CalificacionController.php(16): App\Services\Responsable\CalificacionService->get_actividad_alumno('11')
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\CalificacionController->index()
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\CalificacionController))
#8 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2021-01-02 23:57:54 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `insc`.`...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Responsable\CalificacionModel.php(20): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Models\Responsable\CalificacionModel.php(51): App\Models\Responsable\CalificacionModel->get_inscripciones('11')
#4 C:\xampp\htdocs\comple\app\Services\Responsable\CalificacionService.php(16): App\Models\Responsable\CalificacionModel->get_actividad_alumno('11')
#5 C:\xampp\htdocs\comple\app\Controllers\Responsable\CalificacionController.php(16): App\Services\Responsable\CalificacionService->get_actividad_alumno('11')
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\CalificacionController->index()
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\CalificacionController))
#8 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2021-01-02 23:58:00 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `insc`.`...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Responsable\CalificacionModel.php(20): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Models\Responsable\CalificacionModel.php(51): App\Models\Responsable\CalificacionModel->get_inscripciones('11')
#4 C:\xampp\htdocs\comple\app\Services\Responsable\CalificacionService.php(16): App\Models\Responsable\CalificacionModel->get_actividad_alumno('11')
#5 C:\xampp\htdocs\comple\app\Controllers\Responsable\CalificacionController.php(16): App\Services\Responsable\CalificacionService->get_actividad_alumno('11')
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\CalificacionController->index()
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\CalificacionController))
#8 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2021-01-02 23:58:13 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `insc`.`...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Responsable\CalificacionModel.php(20): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Models\Responsable\CalificacionModel.php(51): App\Models\Responsable\CalificacionModel->get_inscripciones('11')
#4 C:\xampp\htdocs\comple\app\Services\Responsable\CalificacionService.php(16): App\Models\Responsable\CalificacionModel->get_actividad_alumno('11')
#5 C:\xampp\htdocs\comple\app\Controllers\Responsable\CalificacionController.php(16): App\Services\Responsable\CalificacionService->get_actividad_alumno('11')
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\CalificacionController->index()
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\CalificacionController))
#8 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2021-01-02 23:58:31 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `insc`.`...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Responsable\CalificacionModel.php(20): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Models\Responsable\CalificacionModel.php(51): App\Models\Responsable\CalificacionModel->get_inscripciones('11')
#4 C:\xampp\htdocs\comple\app\Services\Responsable\CalificacionService.php(16): App\Models\Responsable\CalificacionModel->get_actividad_alumno('11')
#5 C:\xampp\htdocs\comple\app\Controllers\Responsable\CalificacionController.php(16): App\Services\Responsable\CalificacionService->get_actividad_alumno('11')
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\CalificacionController->index()
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\CalificacionController))
#8 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2021-01-02 23:58:38 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `insc`.`...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Responsable\CalificacionModel.php(20): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Models\Responsable\CalificacionModel.php(51): App\Models\Responsable\CalificacionModel->get_inscripciones('11')
#4 C:\xampp\htdocs\comple\app\Services\Responsable\CalificacionService.php(16): App\Models\Responsable\CalificacionModel->get_actividad_alumno('11')
#5 C:\xampp\htdocs\comple\app\Controllers\Responsable\CalificacionController.php(16): App\Services\Responsable\CalificacionService->get_actividad_alumno('11')
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\CalificacionController->index()
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\CalificacionController))
#8 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2021-01-02 23:58:49 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `insc`.`...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Responsable\CalificacionModel.php(20): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Models\Responsable\CalificacionModel.php(51): App\Models\Responsable\CalificacionModel->get_inscripciones('11')
#4 C:\xampp\htdocs\comple\app\Services\Responsable\CalificacionService.php(16): App\Models\Responsable\CalificacionModel->get_actividad_alumno('11')
#5 C:\xampp\htdocs\comple\app\Controllers\Responsable\CalificacionController.php(16): App\Services\Responsable\CalificacionService->get_actividad_alumno('11')
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\CalificacionController->index()
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\CalificacionController))
#8 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2021-01-02 23:58:52 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `insc`.`...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Responsable\CalificacionModel.php(20): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Models\Responsable\CalificacionModel.php(51): App\Models\Responsable\CalificacionModel->get_inscripciones('11')
#4 C:\xampp\htdocs\comple\app\Services\Responsable\CalificacionService.php(16): App\Models\Responsable\CalificacionModel->get_actividad_alumno('11')
#5 C:\xampp\htdocs\comple\app\Controllers\Responsable\CalificacionController.php(16): App\Services\Responsable\CalificacionService->get_actividad_alumno('11')
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\CalificacionController->index()
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\CalificacionController))
#8 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2021-01-02 23:59:01 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `insc`.`...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Responsable\CalificacionModel.php(20): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Models\Responsable\CalificacionModel.php(51): App\Models\Responsable\CalificacionModel->get_inscripciones('11')
#4 C:\xampp\htdocs\comple\app\Services\Responsable\CalificacionService.php(16): App\Models\Responsable\CalificacionModel->get_actividad_alumno('11')
#5 C:\xampp\htdocs\comple\app\Controllers\Responsable\CalificacionController.php(16): App\Services\Responsable\CalificacionService->get_actividad_alumno('11')
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\CalificacionController->index()
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\CalificacionController))
#8 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2021-01-02 23:59:04 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `insc`.`...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Responsable\CalificacionModel.php(20): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Models\Responsable\CalificacionModel.php(51): App\Models\Responsable\CalificacionModel->get_inscripciones('11')
#4 C:\xampp\htdocs\comple\app\Services\Responsable\CalificacionService.php(16): App\Models\Responsable\CalificacionModel->get_actividad_alumno('11')
#5 C:\xampp\htdocs\comple\app\Controllers\Responsable\CalificacionController.php(16): App\Services\Responsable\CalificacionService->get_actividad_alumno('11')
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\CalificacionController->index()
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\CalificacionController))
#8 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2021-01-02 23:59:09 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `insc`.`...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Responsable\CalificacionModel.php(20): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Models\Responsable\CalificacionModel.php(51): App\Models\Responsable\CalificacionModel->get_inscripciones('11')
#4 C:\xampp\htdocs\comple\app\Services\Responsable\CalificacionService.php(16): App\Models\Responsable\CalificacionModel->get_actividad_alumno('11')
#5 C:\xampp\htdocs\comple\app\Controllers\Responsable\CalificacionController.php(16): App\Services\Responsable\CalificacionService->get_actividad_alumno('11')
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\CalificacionController->index()
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\CalificacionController))
#8 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2021-01-02 23:59:11 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `insc`.`...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Responsable\CalificacionModel.php(20): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Models\Responsable\CalificacionModel.php(51): App\Models\Responsable\CalificacionModel->get_inscripciones('11')
#4 C:\xampp\htdocs\comple\app\Services\Responsable\CalificacionService.php(16): App\Models\Responsable\CalificacionModel->get_actividad_alumno('11')
#5 C:\xampp\htdocs\comple\app\Controllers\Responsable\CalificacionController.php(16): App\Services\Responsable\CalificacionService->get_actividad_alumno('11')
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\CalificacionController->index()
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\CalificacionController))
#8 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2021-01-02 23:59:16 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `insc`.`...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Responsable\CalificacionModel.php(20): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Models\Responsable\CalificacionModel.php(51): App\Models\Responsable\CalificacionModel->get_inscripciones('11')
#4 C:\xampp\htdocs\comple\app\Services\Responsable\CalificacionService.php(16): App\Models\Responsable\CalificacionModel->get_actividad_alumno('11')
#5 C:\xampp\htdocs\comple\app\Controllers\Responsable\CalificacionController.php(16): App\Services\Responsable\CalificacionService->get_actividad_alumno('11')
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\CalificacionController->index()
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\CalificacionController))
#8 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2021-01-02 23:59:31 --> Unable to connect to the database.
#0 C:\xampp\htdocs\comple\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\comple\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT `insc`.`...', Array, false)
#2 C:\xampp\htdocs\comple\app\Models\Responsable\CalificacionModel.php(20): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\comple\app\Models\Responsable\CalificacionModel.php(51): App\Models\Responsable\CalificacionModel->get_inscripciones('11')
#4 C:\xampp\htdocs\comple\app\Services\Responsable\CalificacionService.php(16): App\Models\Responsable\CalificacionModel->get_actividad_alumno('11')
#5 C:\xampp\htdocs\comple\app\Controllers\Responsable\CalificacionController.php(16): App\Services\Responsable\CalificacionService->get_actividad_alumno('11')
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Responsable\CalificacionController->index()
#7 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Responsable\CalificacionController))
#8 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
